master/internal/task/allocation.go

```diff
@@ -66,6 +66,8 @@ type (
 		idleTimeoutWatcher *IdleTimeoutWatcher
 		// proxy state
 		proxies []string
+		// proxyAddress is provided by determined.exec.prep_container if the RM doesn't provide it.
+		proxyAddress *string
```

Comment:
we need to persist this field to the db [so we are able to load it back on master restart].

Reply:
ok, I suppose I need to let you land your pr then.

Reply:
well, I'll hold off until we do 'slurm fault tolerance' or something. is that OK?

Reply:
no problem